sdk/storage/storage-file/src/policies/LoggingPolicy.ts

```diff
@@ -100,7 +103,7 @@ export class LoggingPolicy extends BaseRequestPolicy {
           currentLevel = HttpPipelineLogLevel.WARNING;
           logMessage = `SLOW OPERATION. Duration > ${
             this.loggingOptions.logWarningIfTryOverThreshold
-          } ms. `;
+            } ms. `;
```

Comment:
Shouldn't include code changes. Did you use some auto format tool? I'm using prettier, however looks our format rule doesn't align. You can commit the format rule in another PR.

Reply:
I think you're looking at the changes that are outdated.
Please look at the complete changes :)

And Eng Sys team is working on 'eslint'ing. We'll fix all these formatting issues soon.